templates/default/src/__tests__/pages/_document.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+import Document from '../../pages/_document'
+
+describe('_document', () => {
+  it('renders react html page', () => {
+    const renderer = new ShallowRenderer()
+    const component = renderer.render(<Document />)
+
+    expect(component.props.children.length).toBe(2)
+    expect(component.props.children[1].type).toBe('body')
```

Comment:
further tests would require that I add a displayName in the component manually. I think it may get convoluted so I didn't add. these tests should be sufficient

Reply:
Take a look at the internal project - you can probably copy it and the test case structure (except for the path of the CSS import)

Reply:
@amclin  internal project doesn't test _document.js. the snapshot test that we have in the interal boiler plate maker for _app.js doesn't work the same here because of the way _documentProps works. I had to change render to shallow render. I can copy the test for _app.js once I add that in a different PR.